docs/guide/deployment/oauth-server.md

```diff
@@ -127,8 +127,8 @@ Signer configuration to issue ID/access tokens of OAuth provider for mock testin
 
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
-| `oauthSigner.idTokenKeyFile` | string | `File path to private key for ID token in PEM format.` | `""` |
-| `oauthSigner.accessTokenKeyFile` | string | `File path to private key for access token in PEM format.` | `""` |
-| `oauthSigner.domain` | string | `Audience for access token.` | `""` |
+| `oauthSigner.idTokenKeyFile` | string | `File path to private key for ID token in PEM format which is asymetric RSA key because UI only supports RSA.` | `""` |
+| `oauthSigner.accessTokenKeyFile` | string | `File path to private key for access token in PEM format which is ECDSA key.` | `""` |
```

Comment:
File path to a private ECDSA key in PEM format required for access token signing.